src/main/java/seedu/address/ui/CommandFeatures.java

```diff
@@ -0,0 +1,133 @@
+package seedu.address.ui;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+/**
+ * Type class for tableview
+ */
+public class CommandFeatures {
+
+    private static final CommandFeatures ADDPERSON_COMMAND_FEATURES = new CommandFeatures(
+            "addperson",
+            "Shows a list of all persons in the address book.",
+            "addperson n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]",
+            "addperson n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01"
+            );
+
+    private static final CommandFeatures LISTPERSONS_COMMAND_FEATURES = new CommandFeatures(
+            "listpersons",
+            "Adds a person to the address book.",
+            "listpersons",
+            "listpersons"
```

Comment:
I think there's an error with the description; should be swapped instead?

Reply:
Yes, my mistake. I will update it ASAP.